lxd/api_cluster.go

```diff
@@ -328,25 +327,27 @@ func clusterPut(d *Daemon, r *http.Request) response.Response {
 }
 
 func clusterPutBootstrap(d *Daemon, r *http.Request, req api.ClusterPut) response.Response {
+	s := d.State()
```

Comment:
Just a thought, but rensmjng State() to NewState() would better indicate this function is a factory that generates something new (with cost) rather than just returning an existing entity.

Reply:
Agreed. Not something I want to fix with this PR, but indeed something that'd make sense to fix in the future.

I think we should use the opportunity to audit all our d.State() calls at that point. Limiting to a single call by function and eventually phasing out passing Daemon between functions and instead only passing State.